packages/mui-material-next/src/InputBase/InputBase.test.tsx

```diff
@@ -421,22 +453,22 @@ describe('<InputBase />', () => {
         expect(getByTestId('root')).to.have.class(classes.focused);
 
         act(() => {
-          controlRef.current.onBlur();
+          controlRef.current?.onBlur();
         });
 
         expect(getByTestId('root')).not.to.have.class(classes.focused);
 
         act(() => {
-          controlRef.current.onFocus();
+          controlRef.current?.onFocus();
         });
 
         expect(getByTestId('root')).to.have.class(classes.focused);
       });
 
       it('propagates focused state', () => {
-        function FocusedStateLabel(props) {
-          const { focused } = useFormControl();
-          return <label {...props}>focused: {String(focused)}</label>;
+        function FocusedStateLabel(props: any) {
```

Comment:
Could these `props: any` be typed as `props: { 'data-testid': string, htmlFor: string }` instead? Don't know if it's better, just trying to avoid `any` if possible.